src/oidcCallback.jsx

```diff
@@ -12,7 +14,31 @@ export class oidcCallback extends React.Component {
   }
 
   errorCallback(e) {
-    browserHistory.replace('/')
+    this.props.dispatch(receiveError(e))
+  }
+
+  renderError() {
+    return (
+      <Alert type="error" heading={this.props.error.message}>
+        <div>
```

Comment:
I think this is a good place to use `<React.Fragment>`.

Reply:
I actually did started to do that but my editor didn't like `<>` so I abandoned it out of protest ;)
Also, I wasn't sure it would work due to how we're rendering children inside the Alert, but it actually works fine ^^;